lib/helpers/populate/getModelsMapForPopulate.js

```diff
@@ -17,6 +17,9 @@ const populateModelSymbol = require('../symbols').populateModelSymbol;
 const schemaMixedSymbol = require('../../schema/symbols').schemaMixedSymbol;
 
 module.exports = function getModelsMapForPopulate(model, docs, options) {
+  if (model.schema.base.options.strictPopulate == false && options.strictPopulate == null) {
```

Comment:
`model.schema.base.options.strictPopulate != null`. Why hardcoding false?

Also, `model.base.options` should work and be shorter.

Finally, any way to move this up to the `populate()` function in `model.js` that calls `getModelsMapForPopulate()` and shallow clone the object before modifying? A bit random to modify this option here.